deformable_manipulation_experiment_params/launch/rope_zig_match_params.launch

```diff
@@ -0,0 +1,109 @@
+<launch>
```

Comment:
Whatever comments I made elsewhere, they probably apply here too. Not looking at the whole file.